web-server/src/hooks/useTableSort.ts

```diff
@@ -101,10 +101,44 @@ export const useTableSort = <T = Record<string, any>>(
     return result;
   }, [conf.order, list, simpleSort]);
 
+  const handleCommitSort = useCallback(() => {
+    const result = [...list] as any[];
+    if (!result.some((pr) => pr?.commits)) return simpleSort();
+    if (conf.order === 'asc') {
+      result.sort((a, b) => {
+        return Number(a.commits) - Number(b.commits);
+      });
+    } else {
+      result.sort((b, a) => {
+        return Number(a.commits) - Number(b.commits);
+      });
+    }
+    return result;
+  }, [conf.order, list, simpleSort]);
+
+  const handleLinesSort = useCallback(() => {
+    const result = [...list] as any[];
+    if (!result.some((pr) => pr?.additions)) return simpleSort();
+    if (!result.some((pr) => pr?.deletions)) return simpleSort();
+    if (conf.order === 'asc') {
+      result.sort((a, b) => {
+        return Number(a.additions + a.deletions) - Number(b.additions + b.deletions);
+      });
+    } else {
+      result.sort((b, a) => {
+        return Number(a.additions + a.deletions) - Number(b.additions + b.deletions);
+      });
+    }
+    return result;
+  }, [conf.order, list, simpleSort]);
+  
+
   const sortedList: T[] = useMemo(() => {
     if (conf.field === 'author') return handleAuthorUsernameSort();
     if (conf.field === 'reviewers') return handleReviewerSort();
     if (conf.field === 'first_response_time') return handleResponseTime();
+    if (conf.field === 'commits') return handleCommitSort();
+    if (conf.field === 'additions') return handleLinesSort();
```

Comment:
since you are utilising `handleLinesSort`, they need to be passed into the dependency array of `sortedList`